### DISCUSSION TITLE
Can you specify a custom location for an auto-generated document citation?

### DISCUSSION THREAD

**Discussion by brtarran** [quarto-dev/quarto-cli#6609] [Q&A]
### Description

If you use the `citation: true` option in YAML, an auto-generated bibtext and text citation appears at the end of a Quarto document. I was wondering whether it is possible to specify the location that this citation appears, in the same way that you can use...

```
::: {#refs}
:::
```

...to specify the location that references appear when using a .bib file?

**Comment by cderv** [comment]
This should work as you expect 
https://quarto.org/docs/authoring/footnotes-and-citations.html#bibliography-generation

Did you try this already ? Is this not working ?

**Reply by rgaiacs** [reply]
@brtarran was talking about [Creating Citeable Articles](https://quarto.org/docs/authoring/create-citeable-articles.html). How to move

![appendix-citation](https://github.com/quarto-dev/quarto-cli/assets/1506457/0d5c2195-9d71-4df3-ac1c-59e903dc559e)

(screenshot from https://quarto.org/docs/authoring/images/appendix-citation.png) to the begin of the document.

**Reply by cderv** [reply]
Oh thank you for the screenshot. That help understand. Indeed I misundertood

**Reply by brtarran** [reply]
> @brtarran was talking about [Creating Citeable Articles](https://quarto.org/docs/authoring/create-citeable-articles.html). How to move
> 
> ![appendix-citation](https://github.com/quarto-dev/quarto-cli/assets/1506457/0d5c2195-9d71-4df3-ac1c-59e903dc559e)
> 
> (screenshot from https://quarto.org/docs/authoring/images/appendix-citation.png) to the begin of the document.

Thank you. I should have explained with an example originally. Would have been clearer to all!

**Comment by mcanouil** [comment]
I don't believe there is a way as it is processed as the appendix of the document.

Or you have to use JavaScript for now:

````qmd
---
title: "Quarto Playground"
format: html
citation: true
include-after-body:
  - text: |
      <script type="text/javascript">
      const header = document.querySelector('header');
      const appendixContents = document.querySelector('#quarto-appendix');
      header.insertAdjacentElement('afterend', appendixContents);
      </script>
---

This is a playground for Quarto.
````

**Reply by cderv** [reply]
I was thinking maybe change the style of appendix to plain and move the part around using placeholder in markdown and a Lua filter ? 

@dragonstyle is the customization of automated generation of citation placement something that could be possible ?
